assets/pages/new-contributions-landing/contributionsLandingActions.js

```diff
@@ -264,7 +266,7 @@ const paymentAuthorisationHandlers: PaymentMatrix<(Dispatch<Action>, State, Paym
 };
 
 const onThirdPartyPaymentAuthorised = (paymentAuthorisation: PaymentAuthorisation) =>
-  (dispatch: Dispatch<Action>, getState: () => State): void => {
+  (dispatch: Function, getState: () => State): void => {
```

Comment:
🤔

Reply:
this isn't declared as a `type Action` and so it cannot have type `Dispatch<Action>`.  Should it be included in `type Action`? I saw that a few other places with similar issue used generic Function type which is why I used same fix here.